src/main/java/com/rcv/Logger.java

```diff
@@ -107,7 +107,10 @@ static void setup() throws IOException {
   static void addTabulationFileLogging(String outputPath) throws IOException {
     // create file handler at FINE level (we have detailed audit logging we want to capture)
     // and use our custom formatter
-    tabulationHandler = new FileHandler(outputPath);
+    tabulationHandler = new FileHandler(outputPath,
```

Comment:
A couple issues here:
1. I'm getting log files with .0 and .1 at the end, so they're unrecognized formats. I think you're going to want to use the %g.log thing like we did with the execution logs to avoid this.
2. For some reason I'm getting logs that start in file 1 and end in file 0 when they produce two files...

Reply:
Yeah the %g naming is better, I'll change that one.  for # 2 that may be the expected behavior.  When file 0 gets to 50mb it moves to file 1 and when file 1 gets to 50mb it will move to file 0.

Reply:
Agree that item 2 can occur with the execution log because we're limiting it to only 2 files; but the tabulation log is limited to 1000 files, so it definitely shouldn't go from 1 to 0.

Another minor thing that would be nice to fix (though not sure if it's possible: if there's only 1 file, it shouldn't have 0 or 1 in the filename).

Reply:
Gotcha.  So turns out that IS the expected behavior.  It makes sense when you consider the typical use case of looking at the most recent log file first, and then working your way back in time (https://www.youtube.com/watch?v=vHxFa08XE3w) to find something.  Our use case is the reverse: we have a specific starting point, and want to consume the data from that point forward in time.  We should roll this conversation into the next iteration on audit log format.